Fix a typo (missed space)

quintusm opened the pull request:
# Fix missing space typo

## Summary
Added missing space on line 57

Commit:
Fix a typo (missed space)

Added missing space on line 57

dnfclas commented:
Hi __@quintusm__, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, DNFBOT;

rpetrusha commented on docs/csharp/programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type.md:
```diff
@@ -54,7 +54,7 @@ When you define a class or struct, you decide whether it makes sense to create a
  The implementation details for value equality are different for classes and structs. However, both classes and structs require the same basic steps for implementing equality:  
   
-1.  Override the [virtual](../../../csharp/language-reference/keywords/virtual.md)<xref:System.Object.Equals%28System.Object%29?displayProperty=fullName> method. In most cases, your implementation of `bool Equals( object obj )` should just call into the type-specific `Equals` method that is the implementation of the <xref:System.IEquatable%601?displayProperty=fullName> interface. (See step 2.)  
+1.  Override the [virtual] (../../../csharp/language-reference/keywords/virtual.md)<xref:System.Object.Equals%28System.Object%29?displayProperty=fullName> method. In most cases, your implementation of `bool Equals( object obj )` should just call into the type-specific `Equals` method that is the implementation of the <xref:System.IEquatable%601?displayProperty=fullName> interface. (See step 2.)  
```
@quintusm Thanks for catching the missing space. However, this is the wrong place to add the space. virtual is intended to link to the virtual keyword in the language reference. The space should come between "virtual.md)" and "<xref:". Could you make that change and commit again, please? Then I'll merge your PR. Thanks.

quintusm replied:
I have made the requested change and committed.  Sorry for the mistake.

Commit:
Add missing space

Removed the space from incorrect position and added it to correct position between "virtual.md)" and "<xref:"

rpetrusha commented:
Thanks for fixing this, @quintusm.  I'll merge.
